validator/sawtooth_validator/state/state_view.py

```diff
@@ -32,12 +32,20 @@ def __init__(self, database):
         """
         self._database = database
 
-    def create_view(self, state_root_hash):
+    def create_view(self, state_root_hash=None):
         """Creates a StateView for the given state root hash.
 
+        Args:
+            state_root_hash (str): The state root hash of the state view
+                to return.  If None, returns the state view for the
         Returns:
             StateView: state view locked to the given root hash.
         """
+        # If we are not given a state root hash, then return the default Merkle
+        # database
+        if state_root_hash is None:
```

Comment:
I'll change as I'll assume that there isn't a big performance hit to setting the Merkle root twice every time a state view is created with a non-None `state_root_hash`.